docs/sources/configure-client/profile-types.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Profile types and instrumentation
+menuTitle: Profile types and instrumentation
+description: Learn about the different profiling types available in Pyroscope and
+weight: 100
+aliases:
+  - ../ingest-and-analyze-profile-data/profiling-types/
+  - ../view-and-analyze-profile-data/profiling-types/ # /docs/pyroscope/latest/view-and-analyze-profile-data/profiling-types/
+keywords:
+  - pyroscope
+  - profiling types
+  - application performance
+  - flame graphs
+---
+
+# Profile types and instrumentation
+
+Profiling is an essential tool for understanding and optimizing application performance. In Pyroscope, various profiling types allow for an in-depth analysis of different aspects of your application. This guide explores these types and explain their impact on your program.
+
+Profiling types refer to different dimensions of application performance analysis, focusing on specific aspects like CPU usage, memory allocation, or thread synchronization.
+
+[//]: # 'Shared content for available profile types'
+[//]: # 'This content is located in /pyroscope/docs/sources/shared/available-profile-types.md'
+
+{{< docs/shared source="pyroscope" lookup="available-profile-types.md" version="latest" >}}
+
+Refer to [Understand profiling types and their uses in Pyroscope](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/introduction/profiling-types/) for more details about the profile types.
+
+## Profile type support by instrumentation method
+
+The instrumentation method you use determines which profile types are available. You can use either auto or manual instrumentation.
+
+### Auto-instrumentation with Grafana Alloy
+
+You can send data from your application using Grafana Alloy collector. Alloy supports profiling with eBPF, Java, and Golang in pull mode.
+
+For more information, refer to [Configure the client to send profiles with Grafana Alloy](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/grafana-alloy/).
+
+This table lists the available profile types based on auto instrumentation using Alloy.
+
+| Profile type   | Go (pull) | Java | eBPF (Go) |
```

Comment:
As @bryanhuhta mentioned, should be better just `eBPF` instead of `eBPF (Go)` to be more generic?

Reply:
Good catch! Yes. I have that fix :) It was pushed.

Reply:
oh, you already changed it in d6ea74846f528cc8bf4d101add745c12083411f3 😁